src/fbrs/controller/RecycleBinController.java

```diff
@@ -40,6 +81,49 @@ public void back(ActionEvent actionEvent) {
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
+        refreshTable();
+        searchField.textProperty().addListener((observable, oldValue, newValue) -> search());
+
+        idColumn.setCellValueFactory(new PropertyValueFactory<>("darshKey"));
+        nameColumn.setCellValueFactory(new PropertyValueFactory<>("name"));
+        phoneColumn.setCellValueFactory(new PropertyValueFactory<>("phone"));
+        balanceColumn.setCellValueFactory(new PropertyValueFactory<>("balance"));
+        selectColumn.setCellValueFactory(new PropertyValueFactory<>("selected"));
+
+        CheckBox selectAll = new CheckBox();
+        selectColumn.setGraphic(selectAll);
+        selectAll.setOnAction(this::selectAllBoxes);
+        selectColumn.setCellFactory(CheckBoxTableCell.forTableColumn(selectColumn));
+    }
+
+    private void refreshTable(){
+        ObservableList<User> observableList = FXCollections.observableArrayList();
+
+        users = new FilteredList<>(observableList);
+        observableList.addAll(model.getDeletedUsers());
+
+        table.setItems(users);
+        table.setEditable(true);
```

Comment:
Again...
no need to call table.setEditable(true); every time you refresh
move it to initialize